data_ingestion/app.py

```diff
@@ -61,16 +59,32 @@ def send_points(self, points_by_bucket):
         for bucket in points_by_bucket:
```

Comment:
Lets put this in a try catch

Reply:
This got handled in the `try-catch` on line 86 of this file.

Reply:
We need another try catch because when an exception is raised that try catch will catch it but it will exit the client afterwards. We need another try catch here so that the program doesnt crash but continues on as usual. Also we should log the error.